snap/bin/security-start.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash -e
+
+export KONG_SNAP="${SNAP}/bin/kong-wrapper.sh"
+
+export LOG_DIR=${SNAP_COMMON}/logs
+export CONFIG_DIR=${SNAP_DATA}/config
+export SEC_SEC_STORE_CONFIG_DIR=${CONFIG_DIR}/security-secret-store
+export SEC_GATEWAY_API_CONFIG_DIR=${CONFIG_DIR}/security-gateway-api
+
+# security-secret-store environment variables
+export _VAULT_SCRIPT_DIR=${SNAP}/bin
+export _VAULT_DIR=${SNAP_DATA}/vault
+export _VAULT_SVC=localhost
+export _KONG_SVC=localhost
+export _PKI_SETUP_VAULT_ENV=${SEC_SEC_STORE_CONFIG_DIR}/pki-setup-config-vault.env
+export _PKI_SETUP_KONG_ENV=${SEC_SEC_STORE_CONFIG_DIR}/pki-setup-config-kong.env
+export WATCHDOG_DELAY=10s
+
+# security-gateway-api environment variables
+export KONG_PROXY_ACCESS_LOG=${LOG_DIR}/kong-proxy-access.log
+export KONG_ADMIN_ACCESS_LOG=${LOG_DIR}/kong-admin-access.log
+export KONG_PROXY_ERROR_LOG=${LOG_DIR}/kong-admin-error.log
+export KONG_ADMIN_ERROR_LOG=${LOG_DIR}/kong-admin-error.log
+export KONG_ADMIN_LISTEN="0.0.0.0:8001, 0.0.0.0:8444 ssl"
```

Comment:
Why is 0.0.0.0 used here vs. localhost?

Reply:
This is from the dockerfiles : https://github.com/edgexfoundry/security-secret-store/blob/master/docker-compose-california-0.6.0.yml#L154